en.js

```diff
@@ -1,5 +1,6 @@
 l10n.translations.en = {
   logout: 'Log out',
+  disconnect: 'Disconnect from',
```

Comment:
Discussed with @sharov 

```suggestion
  disconnect: 'Revoke access',
```